nyaa/backend.py

```diff
@@ -29,6 +30,14 @@
     'lpt0', 'lpt1', 'lpt2', 'lpt3', 'lpt4', 'lpt5', 'lpt6', 'lpt7', 'lpt8', 'lpt9',
 ]
 
+# Invalid RSS characters regex, used to sanitize some strings
+ILLEGAL_XML_CHARS_RE = re.compile(u'[\x00-\x08\x0b\x0c\x0e-\x1F\uD800-\uDFFF\uFFFE\uFFFF]')
+
+
+def sanitize_string(string, replacement='?'):
```

Comment:
Could we use `\uFFFD` a.k.a. � here? It may be clearer to see "F��" than to see "F??", so if it's allowed in the XML spec I'd be for it.

Reply:
It's outside the regex, at least.